libraries/azure_backend.rb

```diff
@@ -673,6 +671,10 @@ def method_missing(method_name, *args, &block)
   def respond_to_missing?(*several_variants)
     super
   end
+
+  def to_s
+    "Property is missing! The following are available: #{item.keys.map(&:to_s)}"
```

Comment:
It would be nice to re-word this slightly...
`Property is missing! The following properties are available: #{item.keys.map(&:to_s)}`